### PR TITLE
Fix type tags and todo placement.

### PR DESCRIPTION
**Pull request by marisaleung**
Fixes https://github.com/google/blockly/issues/1260

### CODE REVIEW

File: core/gesture.js [hunk 1]
```diff
@@ -41,8 +41,9 @@ goog.require('goog.math.Coordinate');
 /**
  * NB: In this file "start" refers to touchstart, mousedown, and pointerstart
```
Comment (AnmAtAnm) [inline core/gesture.js]: @rachel-fenichel : What is "NB" suppose to mean, and should this even be a doc comment (`/** ... */`)?

Comment (rachel-fenichel) [inline core/gesture.js]: ["Nota bene"](https://en.wikipedia.org/wiki/Nota_bene), and I could have just used "Note".

No it should not be a doc comment.

Comment (marisaleung) [inline core/gesture.js]: Fixed

File: core/gesture.js [hunk 2]
```diff
@@ -38,11 +38,12 @@ goog.require('goog.asserts');
 goog.require('goog.math.Coordinate');
 
 
-/**
+/*
  * NB: In this file "start" refers to touchstart, mousedown, and pointerstart
```
Comment (AnmAtAnm) [inline core/gesture.js]: For broader audience clarity, replace NB with NOTE:

**Comment by AnmAtAnm** [comment]
Approved, with one nit.

File: core/gesture.js (hunk 2 above)
Comment (marisaleung) [inline core/gesture.js]: Thank you! Done.

**Pull request merged**